cfme/services/requests.py

```diff
@@ -53,234 +365,16 @@ def step(self, *args, **kwargs):
-        RequestException: if multiple matching requests were found
-
-    Returns:
-         The matching :py:class:`cfme.web_ui.Table.Row` if found, ``False`` otherwise.
-    """
-    row = find_request(cells, partial_check)
-    if row.request_state.text in REQUEST_FINISHED_STATES:
-        return row
-    else:
-        return False
-
-
-def debug_requests():
-    logger.debug('Outputting current requests')
-    for page in paginator.pages():
-        for row in fields.request_list.rows():
-            logger.debug(' %s', row)
-
-
-def find_request(cells, partial_check=False):
-    """Finds the request and returns the row element
-
-    Args:
-        cells: Search data for the requests table.
-        partial_check: If to use the ``in`` operator rather than ``==`` in find_rows_by_cells().
-    Returns: row
-    """
-    navigate_to(Request, 'All')
-    for page in paginator.pages():
-        results = fields.request_list.find_rows_by_cells(cells, partial_check)
-        if len(results) == 0:
-            # row not on this page, assume it has yet to appear
-            # it might be nice to add an option to fail at this point
-            continue
-        elif len(results) > 1:
-            raise RequestException(
-                'Multiple requests with matching content found - be more specific!'
-            )
-        else:
-            # found the row!
-            row = results[0]
-            logger.debug(' Request Message: %s', row.last_message.text)
-            return row
-    else:
-        # Request not found at all, can't continue
-        return False
-
-
-def go_to_request(cells, partial_check=False):
-    """Finds the request and opens the page
-
-    See :py:func:`wait_for_request` for futher details.
-
-    Args:
-        cells: Search data for the requests table.
-    Returns: Success of the action.
-    """
-    row = find_request(cells, partial_check)
-    if row:
-        sel.click(row)
-        return True
-    else:
-        return False
-
-
-@contextmanager
-def copy_request(cells):
-    """Context manager that opens the request for editing and saves or cancels depending on success.
-
-    Args:
-        cells: Search data for the requests table.
-    """
-    if not go_to_request(cells):
-        raise Exception("The requst specified by {} not found!".format(str(cells)))
-    sel.wait_for_element(buttons.copy)  # It is glitching here ...
-    sel.click(buttons.copy)
-    try:
-        yield
-    except Exception:
-        sel.click(buttons.cancel)
-        raise
-    else:
-        from cfme.provisioning import provisioning_form
-        btn = provisioning_form.submit_copy_button
-        sel.wait_for_element(btn)
-        sel.click(btn)
-        flash.assert_no_errors()
-
-
-@contextmanager
-def edit_request(cells):
-    """Context manager that opens the request for editing and saves or cancels depending on success.
+    def am_i_here(self, *args, **kwargs):
+        return match_page(summary=self.obj.description)
```

Comment:
If we could get rid of match_page, that would be cool, since that one uses pytest_selenium :)